docs/usage_guide.rst

```diff
@@ -206,15 +206,18 @@ Rotki can import any trade CSV data exported from cointracking.info. But in gene
 Tracking accounts and balances
 **********************************
 
-To track blockchain accounts&balances and also FIAT balances you need to visit the "Accounts/balances" section from the left sidebar.
+To manage Accounts & Balances (Blockchain Balances, Exchange Balances, and Manual Balances including fiat) you need to visit the "Accounts & Balances" section from the left sidebar.
```

Comment:
Perhaps add something here, either here or in its own subsection that shows you can also check exchange balances in that page.

Reply:
Good call! Added in next commit.